src/components/MessageList.vue

```diff
@@ -421,6 +427,16 @@ export default {
                 this.$el.scrollTop = this.$el.scrollHeight;
             }
         },
+        maybeScrollToId(id) {
+            let msgs = this.$el.getElementsByClassName('kiwi-messagelist-message');
```

Comment:
This would be easier using `this.$el.querySelector('.kiwi-messagelist-message[data-message-id="'+theid+'"]`